docs/modules/ROOT/pages/configuration/global-filters.adoc

```diff
@@ -0,0 +1,48 @@
+= Configuring Global Filters on KubeArchive
+
+KubeArchive allows cluster owners to configure a set of filters that are applied
+to any KubeArchive-enabled namespace. To configure them follow the next steps:
```

Comment:
What does "KubeArchive-enabled namespace" mean? Should the user set up specific permissions to the namespace they want to be managed by a global workspace?

Reply:
Hmm.. I tried to come up with a fancy word for "namespace where you already have a KubeArchiveConfig", but since its not mentioned anywhere else I can use the full phrase. Any other ideas?

Reply:
Maybe we could go with the same approach than with the acronyms: stating at the begining what "KubeArchive-enabled namespace" means so we can refer to it in the rest of the page.

Something like:

```To get a namespaced managed by a global filter the namespace must contain a KubeArchiveConfig. This CR could have the following placeholder values if no aditional archiving logic need to be provided:

<insert a KAC CR with placeholder values>

From now on we will refer to these namespaces as "KubeArchive-enabled namespaces".
```

Reply:
We don't use it here that much, I opted for the phrase.

Reply:
I am ok with that. But what about showing an example of a dummy KubeArchiveConfig resource so that the namespace take the configuration of the global filter without adding new filters?
For that case could this work?

```yaml
---
apiVersion: kubearchive.kubearchive.org/v1alpha1
kind: KubeArchiveConfig
metadata:
  name: kubearchive
  namespace: ns
spec:
  resources: []
```

Reply:
I see - https://github.com/kubearchive/kubearchive/issues/526

Reply:
I think this works:
```---
apiVersion: kubearchive.kubearchive.org/v1alpha1
kind: KubeArchiveConfig
metadata:
  name: kubearchive
  namespace: ns
spec:
  ```
  The `resources` are optional.

Reply:
Ah, I see the bug in #526. We still need to create the global filters in this case.

Reply:
> That does not work currently, we have a story to solve it.

From the users perspective this may appear to be working!

Reply:
> "any namespace where you already have a KubeArchiveConfig"

I would suggest "any namespace containing a `KubeArchiveConfig` custom resource".